include/netpacket/if_addr.h

```diff
@@ -0,0 +1,110 @@
+/****************************************************************************
+ * include/netpacket/if_addr.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_NETPACKET_IF_ADDR_H
+#define __INCLUDE_NETPACKET_IF_ADDR_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <sys/types.h>
+#include <netpacket/netlink.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define IFA_MAX (__IFA_MAX - 1)
+
+/* ifa_flags */
+#define IFA_F_SECONDARY       0x01
+#define IFA_F_TEMPORARY       IFA_F_SECONDARY
+
+#define IFA_F_NODAD           0x02
+#define IFA_F_OPTIMISTIC      0x04
+#define IFA_F_DADFAILED       0x08
+#define IFA_F_HOMEADDRESS     0x10
+#define IFA_F_DEPRECATED      0x20
+#define IFA_F_TENTATIVE       0x40
+#define IFA_F_PERMANENT       0x80
+#define IFA_F_MANAGETEMPADDR 0x100
+#define IFA_F_NOPREFIXROUTE  0x200
+#define IFA_F_MCAUTOJOIN     0x400
+#define IFA_F_STABLE_PRIVACY 0x800
+
+/* backwards compatibility for userspace */
+#define IFA_RTA(r)  ((FAR struct rtattr *)(((FAR char *)(r)) + \
```

Comment:
All comments should be followed by a blank line.  Not detected by nxstyle?

Reply:
Because it's a macro?

Reply:
> 
> 
> Because it's a macro?

That is probably why nxstyle failed to detect the coding standard violation for some reason.

Reply:
Update for resolve the style issue